sound/soc/sof/loader.c

```diff
@@ -359,8 +360,9 @@ static int snd_sof_fw_ext_man_parse(struct snd_sof_dev *sdev,
 /*
  * IPC Firmware ready.
  */
-static void sof_get_windows(struct snd_sof_dev *sdev, int bar)
+static void sof_get_windows(struct snd_sof_dev *sdev)
 {
+	int bar = snd_sof_dsp_get_bar_index(sdev, SOF_FW_BLK_TYPE_SRAM);
```

Comment:
this seems to exactly undo the previous patch in the series with the only exception of not re-adding the error handling. Can we simply remove the previous patch?

Reply:
Good point, let me see how it will change the code, it should be straight forward to drop that patch.

Reply:
Done